auth/opa/rpcauth/rpcauth.go

```diff
@@ -106,7 +107,17 @@ func (g *Authorizer) Eval(ctx context.Context, input *RPCAuthInput) error {
 	}
 	logger.V(1).Info("authz policy evaluation result", "authorizationResult", result, "input", input)
```

Comment:
Let's log the denial hints. Maybe add an else clause to the if statement, and then add this log to both conditions?
e.g.
```
if result {
  logger.V(1).Info("request approved", "input", input)
} else { // denied
  hints := ...
  logger.V(1).Info("request denied", "input", input, "denialHints", hints)
}
```

Reply:
I've changed the log to include the hints.

I decided to keep a single log line instead of separate log lines for request approved vs denied. It'll leave an empty denialHints on success, but I think that's okay. I think it's useful to use the same log line for both success and failure.